heron/storm/src/java/BUILD

```diff
@@ -8,21 +10,29 @@ storm_deps_files = [
     "//third_party/java:kryo",
 ]
 
+storm_deps_files_nl = [
+    "//heron/api/src/java:api-java-neverlink",
+    "//heron/common/src/java:basics-java",
+    "//heron/simulator/src/java:simulator-java-neverlink",
```

Comment:
This sounds like a better move, but I feel like that needs more discussion, planning, and effort for implementation.